src/renderer.ts

```diff
@@ -782,11 +1287,19 @@ export const renderers: {[key: string]: ICellRendererFactory} = {
   min: combineCellRenderer,
   mean: combineCellRenderer,
   script: combineCellRenderer,
+  heatmapcustom: new HeatmapCellRenderer(),
```

Comment:
try finding a better name that `heatmapcustom` the `custom` suffix isn't a useful name for a default renderer of a library